### ISSUE TITLE
Create a documentation page to list all the projects using Asciidoctor.js

### ISSUE THREAD

**Issue by ggrossetie** [asciidoctor/asciidoctor.js#607]

**Comment by HillLiu** [comment]
Just Share a React  Asciidoctor.JS component. 
https://github.com/react-atomic/react-atomic-organism/tree/master/packages/organism-react-asciidoc/

**Comment by mojavelinux** [comment]
I would caution against "all the projects using Asciidoctor.js" as that's exhausting and can get you into trouble. It would be better to highlight notable uses of Asciidoctor.js.

If you want, you can point to GitHub's Network page to show all projects on GitHub that depend on it. See https://github.com/asciidoctor/asciidoctor.js/network/dependents.

**Comment by ggrossetie** [comment]
Yes, I tend to agree. I created that kind of page for Kroki, and it can definitely become exhausting and potentially lead to problems.
